plugin/example/source/dextool/plugin/cpptestdouble/runner.d

```diff
@@ -0,0 +1,150 @@
+/**
+Copyright: Copyright (c) 2017, Joakim Brännström. All rights reserved.
+License: MPL-2
+Author: Joakim Brännström (joakim.brannstrom@gmx.com)
+
+This Source Code Form is subject to the terms of the Mozilla Public License,
+v.2.0. If a copy of the MPL was not distributed with this file, You can obtain
+one at http://mozilla.org/MPL/2.0/.
+
+This file contains an example plugin.
```

Comment:
I think it can be good to add a brief description of what the plugin does. Something that makes the reader think about a problem that will be solved etc.